src/libcore/intrinsics.rs

```diff
@@ -44,6 +44,10 @@
 
 use marker::Sized;
 
+#[cfg(stage0)] pub use self::copy_memory as copy;
```

Comment:
The string `stage0` is enough, I regularly clean out via `git grep stage0` whenever a new snapshot is registered. We have very little permanent `#[cfg(stage0)]`